obal/data/playbooks/nightly/metadata.obal.yaml

```diff
@@ -3,3 +3,11 @@ help: >
 
 
   This typically uses unreleased sources taken from a git repository.
+variables:
+  nightly_sourcefiles:
+    help: Comma-separated list of absolute filepaths to be copied and used for RPM sources.
```

Comment:
Is the native format also comma separated? Might be good to investigate moving to an array but that means this also needs to change so :+1: to start with this

Reply:
I don't think there's stuff currently in our help mechanisms for arrays. but you're right, i'd need to change the `nightly` playbook a bit also.

Reply:
We now have `action: append` which allows you to repeat the argument, in case you want to modify the playbook.